lib/mongo/operation/write/command/insert.rb

```diff
@@ -40,11 +40,14 @@ class Insert
           #
           # @since 2.0.0
           def selector
-            { insert: coll_name,
-              documents: documents,
-              writeConcern: write_concern.options,
-              ordered: ordered?
+            cmd = {
+                insert: coll_name,
+                documents: documents,
+                writeConcern: write_concern.options,
+                ordered: ordered?
             }
+            cmd['bypassDocumentValidation'] = true if bypass_document_validation
+            return cmd
```

Comment:
You don't need to write "return"
Ruby methods return the result of the last expression evaluated. So you can just write

```ruby
cmd
```

However, I would write the last line as it is in the following:

```ruby
def selector
  cmd = {
    insert: coll_name,
    documents: documents,
    writeConcern: write_concern.options,
    ordered: ordered?
    }
  bypass_document_validation ? cmd.merge!(BYPASS_DOCUMENT_VALIDATION => true) : cmd
end
```

In the above, I'm reusing the constant BYPASS_DOCUMENT_VALIDATION so I'm not instantiating another String. I'm also using Hash#merge! instead of explicitly setting the hash value because it returns the hash as a result of calling #merge. The code is more succinct this way.